Makefile

```diff
@@ -3,7 +3,7 @@
 #### Constant variables
 export STACKSLIST ?= incubator/nodejs
 # use -count=1 to disable cache and -p=1 to stream output live
-GO_TEST_COMMAND := go test -v -count=1 -p=5 -parallel 5 -covermode=count -coverprofile=cover.out -coverpkg ./cmd -timeout 15m
+GO_TEST_COMMAND := go test -v -count=1 -p=2 -parallel 5 -covermode=count -coverprofile=cover.out -coverpkg ./cmd
```

Comment:
We will need to look more into this after merging

Reply:
I tested this change on master branch and compared how long the testing run takes to complete with `-p=5` and `-p=2` and there was no clear difference between the two. Both took a similar enough amount of time that I don't think it made any (at least _substantial_) difference in reducing it; but yeah, I'd prefer to investigate it officially a bit more.